glog.py

```diff
@@ -102,7 +126,7 @@ def format(self, record):
             level = '?'
         date = time.localtime(record.created)
         date_usec = (record.created - int(record.created)) * 1e6
-        record_message = '%c%02d%02d %02d:%02d:%02d.%06d %s %s:%d] %s' % (
+        record_message = '[%c%02d%02d %02d:%02d:%02d.%06d %s %s:%d] %s' % (
```

Comment:
The opening bracket was a mistake... probably result of mindless bracket pairing when touching code when sleepy.  Fixed it after I sent you the PR.